src/config/fbInit.js

```diff
@@ -9,10 +9,13 @@ import config from "./fbConfig.js";
 
 // Init Firebase
 firebase.initializeApp(config);
-// Init analytics
-if (config.measurementId) {
+/**
```

Comment:
Is it wanted that you use /** */ as header? Because every other header starts with //>.